docs/install/containers.md

```diff
@@ -144,19 +149,21 @@ To upgrade Harvest:
    This is needed since the new version may contain new templates, dashboards, or other files not included in the Docker
    image.
 
-2. Copy your existing `harvest.yml` into the new Harvest directory created in step #1.
+2. [Stop all containers](#stop-all-containers). If you change directories, be sure to shut down any running Harvest containers from the previous directory.
```

Comment:
We should change the `Stop all containers` section to include more words about this. Something like this:

> If you see this error [insert error] when trying to stop your containers, it is probably because you are running `docker-compose down` from a different directory than `up` was run. You can still stop and remove the old containers by running: `docker ps -qa | xargs docker stop | xargs docker rm` Note: This will stop and remove all running containers including any non-Harvest ones.